utils/getRepoURL.go

```diff
@@ -22,8 +23,18 @@ func GetRepoURL() string {
 		}
 	}
 	if repo == "" {
-		// logger.ErrorLogger.Println("No Repo Url Provided")
-		spinner.LogMessage("No Repo Url Provided", "fatal")
+		// Get repo name from git config file
+		out, err := exec.Command("git", "config", "--get", "remote.origin.url").Output()
+		if err != nil {
+			spinner.LogMessage("Can't get repo url from .git/config file: "+err.Error(), "fatal")
```

Comment:
@KatieHarris2397 this doesn't need to be fatal if you can't get the repo url just add the message you can't find it ask them to add it to the builder.yaml and add an empty string as the value

Reply:
Agree